code/web/src/modules/user/Profile.js

```diff
@@ -5,13 +5,17 @@ import { connect } from 'react-redux'
 import { Helmet } from 'react-helmet'
 import { Link } from 'react-router-dom'
 
+
 // UI Imports
+import ImageTile from '../../ui/image/Tile'
 import { Grid, GridCell } from '../../ui/grid'
 import { H3, H4 } from '../../ui/typography'
 import Button from '../../ui/button'
 import { grey, grey2 } from '../../ui/common/colors'
+import { level1 } from '../../ui/common/shadows'
```

Comment:
This is an import I could understand better. But I do currenlty at least see the utility of.